src/env.cc.in

```diff
@@ -40,7 +40,7 @@ const std::string Env::GetInstallPath() {
   if (instdir_.length() > 0)
     return instdir_;
   std::string instdir = GetEnv("_");
-  if (instdir.length() > 0)
+  if (instdir.length() <= 0)
```

Comment:
This seems like a big deal?!?!

Reply:
Right? The `_` env var (and thus `instdir`) should be the path to the executable.  If that env var is nonexistent then we use the path passed in by CMake, otherwise we use the location of the running executable.  This was causing cyclus to always look in `/root/.local` for things even if it is installed elsewhere.  Pretty sure the issue manifested itself when I installed the deb on a fresh machine it couldn't find the rng schema located in the `share` directory

Reply:
Actually I don't think this fix is robust enough.  I've opened an issue #1688

However,  #1688 should not block this PR in my opinion